include/lmms_math.h

```diff
@@ -27,12 +27,13 @@
 
 #include <QtGlobal>
 #include <algorithm>
+#include <cassert>
```

Comment:
Is there a reason you added the assert header, I don't see any asserts in this commit.

Reply:
Nvm, didn't see that you removed the header previously.